pages/lobby_page.py

```diff
@@ -244,3 +245,28 @@ def delete_ico(self):
     def click_delete_ico(self):
         for delete in self.delete_ico():
             delete.click()
+
+    def invite_team_form(self):
+        self.context.wait.until(lambda driver: driver.find_element_by_xpath('//a[text()="Invite your team"]'))
```

Comment:
Have you try EC waits? They didn't work here? As I understand from our course firstly we should to try use EC waits, and if, for some reasons, wait don't work correct - we can use other type of waits.

Reply:
Done.